src/js/modules/sliderOurVisitors.js

```diff
@@ -124,4 +124,21 @@ function selectNextBullet() {
 	bulletCurrentPosition++;
 	bullets[bulletCurrentPosition].classList.add("active");
 	activeBullet = bullets[bulletCurrentPosition];
+}
+
```

Comment:
I see approximately the same code in all the places, make separate helper for it and import to sliders

Reply:
Done.